tests/e2e/specs/dashboard/integrations/sitekit.js

```diff
@@ -47,7 +49,7 @@ describe('Site Kit integration with dashboard', () => {
       text: 'Editor Settings',
     });
 
-    await percySnapshot(page, 'Stories Dashboard with Site Kit');
+    await percySnapshot(page, 'Stories Dashboard with Site Kit', { percyCSS });
```

Comment:
Instead of adding this class name, can't we just do

```js
await expect(page).not.toMatch('Loading…'); // Wait until media has loaded.
await percySnapshot(page, 'Stories Dashboard with Site Kit');
```

Or perhaps

```js
await page.waitForResponse(response => response.url().includes('web-stories/v1/media')); // Wait until media has loaded.
await percySnapshot(page, 'Stories Dashboard with Site Kit');
```

Could even do both just to be safe:

```js
// Wait until media has loaded.
await page.waitForResponse(response => response.url().includes('web-stories/v1/media'));
await expect(page).not.toMatch('Loading…');

await percySnapshot(page, 'Stories Dashboard with Site Kit');
```

Reply:
Fixed https://github.com/google/web-stories-wp/pull/5571/commits/46b2c73f51a94473e89da28de7bdd22a4fa9f152